### PR TITLE
Add run_as setting to Add new API view

### PR DESCRIPTION
**Pull request by Desvelao**
HI team, this PR resolves:
- add `run_as` setting to Add new API view

Closes: #3017

### CODE REVIEW

File: public/components/settings/api/add-api.js [hunk 1]
```diff
@@ -150,7 +151,8 @@ export class AddApi extends Component {
           <EuiCode>{'<api_port>'}</EuiCode> is the port,{' '}
           <EuiCode>{'<api_username>'}</EuiCode> and{' '}
           <EuiCode>{'<api_password>'}</EuiCode> are the credentials to
-          authenticate.
+          authenticate,{' '}
+          <EuiCode>{'<run_as>'}</EuiCode> defines if the app user permissions depends on the authentication context (<EuiCode>{'true'}</EuiCode> / <EuiCode>{'false'}</EuiCode>).
```
Comment (frankeros) [inline public/components/settings/api/add-api.js:155]: spell correction
` defines if the app user's permissions...`

**Review by frankeros — COMMENTED** [review]
Tested local :heavy_check_mark: 

---

just a spell correction

File: public/components/settings/api/add-api.js (hunk 1 above)
Comment (Desvelao) [reply public/components/settings/api/add-api.js:155]: 👍

**Review by frankeros — APPROVED** [review]
LGTM! :tada: 
Good job!

**Pull request merged**